apps/auth/src/app/sign-in/sign-in.component.html

```diff
@@ -1,7 +1,54 @@
-<p>sign-in works!</p>
-<button (click)="onSignInViaEmail()">Sign in oooo</button>
-<a [routerLink]="['/sign-up']">Visit sign up</a>
-
-<p>{{ isLoading$ | async }}</p>
-<p>{{ error$ | async | json }}</p>
-<p>{{ allState$ | async | json }}</p>
+<div class="tui-container_fullwidth main-container">
```

Comment:
Extract the form into it's own component....look at the sign up for reference.

Also I want you to extract the sign in/sign up via google into a single button component with all the required functionality

Reply:
I will take care that